### PR TITLE
Merge `feature-associations` to `development` branch

### PR DESCRIPTION
**Pull request by o-rashad**
The following have been added to the project : 
- Create a class Classroom
- Create the `has-many/belongs-to` relationship between Classroom and Student. 
- Create a class Book
- Create a class Rental
- Create the `many-to-many` `(also has-many-through)` relationship between Person and Book using the intermediate class Rental.

### CODE REVIEW

**Review by carlos-ssh — APPROVED** [review]
Hi @od-c0d3r , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
```
Comment (carlos-ssh) [inline book.rb:10]: [OPTIONAL]
- Update your ``README`` file.

Comment (o-rashad) [reply book.rb:10]: I will work on the README file ASAP 👍🏼

**Comment by o-rashad** [comment]
@carlos-ssh Thank you for the review 😄🚀

**Pull request merged**